frontend/src/utils/address.ts

```diff
@@ -116,21 +122,35 @@ export const isAddressSafe = async (name: string, userAddress: string, stealthAd
   if (destinationAddress.toLowerCase() === stealthAddress.toLowerCase()) reasons.push(`${tc('Utils.Address.it')} ${isDomain ? tc('Utils.Address.resolves-to') : tc('Utils.Address.is')} ${tc('Utils.Address.same-addr-as-stealth')}`); // prettier-ignore
 
   // Check if address owns any POAPs
-  if (await hasPOAPs(destinationAddress)) reasons.push(`${isDomain ? tc('Utils.Address.address-it-resolves-to') : tc('Utils.Address.it')} ${tc('Utils.Address.has-poap-tokens')}`); // prettier-ignore
+  const hasPOAPsCheck = async () => {
+    const has = await hasPOAPs(destinationAddress);
```

Comment:
FYI: This was erroring locally. I think the domain changed.

https://documentation.poap.tech/reference/getactionsscan-5

Reply:
Thanks for catching that, tracking in https://github.com/ScopeLift/umbra-protocol/issues/467